pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -5599,6 +5599,33 @@ See http://developers.google.com/maps/documentation/utilities/polylinealgorithm`
 			},
 		},
 	),
+	"st_transscale": makeBuiltin(defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"geometry", types.Geometry},
+				{"deltaX", types.Float},
```

Comment:
these should be camel case, i.e. `delta_x`, `delta_y`, `x_factor`, `y_factor`

Reply:
noted